### PR TITLE
UPDATE PAYROLL SECTION

### PR DESCRIPTION
**Pull request by shawnaym**
Update to include change of hands from Jeff Ho to Shawna Young-Marcelino and reflect additional Payroll AORs that weren't previously captured

### CODE REVIEW

File: operations/finance/operations-playbook.md [hunk 1]
```diff
@@ -103,11 +103,16 @@ Internal processes and framework for day-to-day payroll processes.
...
-|[Deel walkthrough](https://docs.google.com/document/d/1ZgHUD1B9KIzBQAZiHfL7yvYUBQ4lDQvvTgG-JF6kbw8/edit#)|`Deel`|Process Narrative| |bi-monthly|Ben Graham|Accounting|
-|[US Payroll](https://docs.google.com/document/d/1TlDpPMZqs_sq1k2bjTxIkcBtff1y52Reeo5h3UM0IAY/edit?usp=sharing)|`Netsuite` `TriNet`|Playbook| |Weekly|Jeff Ho|Payroll|
-|[International Payroll](https://docs.google.com/document/d/1OJ_JoiSdA84h-IN6UZ9O6ZkzW2wgGg2Mv5047q1gKxQ/edit?usp=sharing)|`Netsuite` `Vistra`|Playbook| |Monthly|Jeff Ho, Vistra|Payroll|
-|[401K Playbook](https://docs.google.com/document/d/1Ny0zOr98InPnybaQoMP82laXPmtNDTkXuECIBfyVCx8/edit#heading=h.s1shpwm3yp4j)|`Netsuite` `Ubiquity`|Playbook| |Weekly|Jeff Ho|Payroll|
-|[Deel payroll](https://docs.google.com/document/d/1I3OzopX0tuXB0vvKts3fFB4jke6SLFWrBi9uJL69tOw/edit#)|`Deel`|Playbook| |Weekly|Jeff Ho|Accounting|
+|[Deel walkthrough](https://docs.google.com/document/d/1ZgHUD1B9KIzBQAZiHfL7yvYUBQ4lDQvvTgG-JF6kbw8/edit#)|`Deel`|Process Narrative| |semi-monthly|Ben Graham|Accounting|
+|[Deel payroll](https://docs.google.com/document/d/1I3OzopX0tuXB0vvKts3fFB4jke6SLFWrBi9uJL69tOw/edit#)|`Deel`|Playbook| |Monthly|Shawna Young-Marcelino|Accounting|
```
Comment (TQuock) [inline operations/finance/operations-playbook.md]:
```suggestion
|[Deel Payroll](https://docs.google.com/document/d/1I3OzopX0tuXB0vvKts3fFB4jke6SLFWrBi9uJL69tOw/edit#)|`Deel`|Playbook| |Monthly|Shawna Young-Marcelino|Accounting|
```

**Review by TQuock — COMMENTED** [review]
Thanks for making these updates, Shawna! Love seeing your name on the board. 

A few Notes:
- I am getting 404 notices from some of the updated links. Can you check to make sure we have the right links. The few I noticed are Contractor Invoice Approval, Commission Approval and Payment. There may be others
- I looked into the payroll/accounting folders for those documents we labeled access rights fo and didn't see every document listed in the Payroll section. Can you do a double-check to ensure all documents are sitting in the correct permissions folders in GSuite?

**Review by justinegeffen — APPROVED** [review]

**Comment by shawnaym** [comment]
There should be no links included for Commission and contract invoice approval as we do not have any process walk through guides for these. Ben and I discussed and don't think that we would share these if they existed either just due to the sensitivity

I will review the walk throughs to ensure the payroll links are correct though :) 

Justine, I don't see these changes reflected yet. Is there something else we need to take these live?

**Comment by cwarnermm** [comment]
@shawnaym - I've merged your changes into the Handbook and they should be live within minutes. Please review to ensure links work as expected.

**Pull request merged**